src/components/editor/inputs/InputLang.jsx

```diff
@@ -96,12 +99,10 @@ const InputLang = (props) => {
               <input
                 type="radio"
                 className="form-check-input"
-                name="lang"
-                id={`noLangRadio-${props.textValue}`}
-                data-testid={`noLangRadio-${props.textValue}`}
+                id="absent"
                 value="absent"
-                defaultChecked={!langPresent}
-                onChange={disableLanguageSelector}
+                checked={radioButtonValue === "absent"}
+                onChange={handleLanguageRadio}
               />
```

Comment:
1. we now control if the radio button is checked or not based on the current value in state
2. our new function to handle when the user clicks the radio button

Reply:
much much clearer